app/pages/api/auth/[...nextauth].ts

```diff
@@ -23,6 +23,19 @@ import CognitoProvider from "next-auth/providers/cognito"
 import SlackProvider from "next-auth/providers/slack"
 import Okta from "next-auth/providers/okta"
 import AzureB2C from "next-auth/providers/azure-ad-b2c"
+import AppleProvider from "next-auth/providers/apple"
+
+// // Uncomment and copy log to .env.local every 6 months
+// import { generateClientSecret } from "next-auth/providers/apple"
+// const appleSecret = await generateClientSecret({
+//   clientId: process.env.APPLE_ID,
+//   keyId: process.env.APPLE_KEY_ID,
+//   teamId: process.env.APPLE_TEAM_ID,
+//   privateKey: process.env.APPLE_PRIVATE_KEY,
+// })
+// console.log(
+//   `APPLE_SECRET=${appleSecret} # Created: ${new Date().toISOString()} Update if more than 6 months old.`
+// )
```

Comment:
I think we could move this to the [`README`](https://github.com/nextauthjs/next-auth/blob/main/app/README.md) within the example app folder.